### PR TITLE
Add package attributes from template `package.json` to new project's `Ballerina.toml`

### PR DESCRIPTION
**Pull request by pramodya1994**
Fixes https://github.com/ballerina-platform/ballerina-lang/issues/34807

## Check List 
- [ ] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [x] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/CommandUtil.java [hunk 1]
```diff
@@ -347,10 +347,18 @@ public static void writeBallerinaToml(Path balTomlPath, PackageJson packageJson,
...
         }
 
-        Files.writeString(balTomlPath, "\nexport = [" + stringJoiner.toString() + "]"
+        Files.writeString(balTomlPath, "\nexport = [" + stringJoiner + "]"
                 .replaceFirst(packageJson.getName(), packageName), StandardOpenOption.APPEND);
-        Files.writeString(balTomlPath, "\nballerina_version = \"" + packageJson.getBallerinaVersion()
+        Files.writeString(balTomlPath, "\ndistribution = \"" + packageJson.getBallerinaVersion()
                 + "\"", StandardOpenOption.APPEND);
+
+        writePackageAttributeArray(balTomlPath, packageJson.getLicenses(), "license");
```
Comment (azinneera) [inline cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/CommandUtil.java:355]: We allow additional properties under `[package]`. Shall we blindly copy all the elements in the package.json without copying only the known atttributes?

Comment (pramodya1994) [reply cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/CommandUtil.java:355]: We are adding properties to `package.json` inside [`addPackageJson`](https://github.com/ballerina-platform/ballerina-lang/blob/master/compiler/ballerina-lang/src/main/java/io/ballerina/projects/BalaWriter.java#L141) method. Here we don't add user-defined properties to `package.json` yet. So we can't add these ATM.

Comment (azinneera) [reply cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/CommandUtil.java:355]: I think the users would need to see the undefined ones also in templates. We can parse to a generic object and get the remaining values.

We can still keep the part that parses to package json to validate the balais compatible but when extracting fields, lets use a generic json object and write all key-value pairs

Comment (azinneera) [reply cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/CommandUtil.java:355]: Sorry I misunderstood. Just realized the additional properties are not saved in the bala file. We can go ahead with this impl.

**Comment by azinneera** [comment]
The pr should be opened to `stage-slbeta6.x` branch.

**Pull request closed without merging**